### PR TITLE
Change the story: add doctor and pills

### PR DESCRIPTION
**Pull request by ScarletSnail**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1698)

### CODE REVIEW

**Comment by ScarletSnail** [comment]
Please review this, @casey-collab

File: ukrainian/call-a-doctor/doctor.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+З'являється лікар і пропонує Вам з'їсти синю або червону зефірку.
+
+[З'їм синю зефірку, просто випустіть мене звідси!](call-a-doctor/pills/blue-pill.md)
```
Comment (casey-collab) [inline ukrainian/call-a-doctor/doctor.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: ukrainian/call-a-doctor/doctor.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+[З'їм червону зефірку і дізнаюся усю правду про дивний зефірний світ](call-a-doctor/pills/red-pill.md)
```
Comment (casey-collab) [inline ukrainian/call-a-doctor/doctor.md]: This link is also invalid.

File: ukrainian/call-a-doctor/pills/blue-pill.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+З'являється старенький дідок, який подає Вам синю зефірку і стакан теплого какао.
```
Comment (casey-collab) [inline ukrainian/call-a-doctor/pills/blue-pill.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: ukrainian/call-a-doctor/pills/blue-pill.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+
+[Ви з'їдаєте все, втрачаєте свідомість, і ...](marshmallow.md)
```
Comment (casey-collab) [inline ukrainian/call-a-doctor/pills/blue-pill.md]: This link is also invalid.

File: ukrainian/call-a-doctor/pills/red-pill.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+Раптово Ви усвідомлюєте усю красу зефірного світу і вирішуєте самі стати зефіркою.
```
Comment (casey-collab) [inline ukrainian/call-a-doctor/pills/red-pill.md]: This file has also been added to a directory that already contained a page.

File: ukrainian/call-a-doctor/pills/red-pill.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+
+Після болісного перетворення, ви стаєте частиною дивної зефірної кімнати,
+і ось перед Вами [людина, яка має зробити свій вибір](marshmallow.md)
```
Comment (casey-collab) [inline ukrainian/call-a-doctor/pills/red-pill.md]: This link is also invalid.

File: ukrainian/call-a-doctor/pills/blue-pill.md (hunk 3 above)
Comment (casey-collab) [inline ukrainian/call-a-doctor/pills/blue-pill.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: ukrainian/call-a-doctor/pills/red-pill.md (hunk 5 above)
Comment (casey-collab) [inline ukrainian/call-a-doctor/pills/red-pill.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ScarletSnail** [comment]
Please review this, @casey-collab

File: ukrainian/call-a-doctor/pills/blue-pill.md (hunk 3 above)
Comment (casey-collab) [inline ukrainian/call-a-doctor/pills/blue-pill.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: ukrainian/call-a-doctor/pills/red-pill.md (hunk 5 above)
Comment (casey-collab) [inline ukrainian/call-a-doctor/pills/red-pill.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ScarletSnail** [comment]
Thanks

**Comment by ScarletSnail** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :star:

**Pull request merged**